plugins/split_text_to_size.js

```diff
@@ -25,301 +25,326 @@
  * ====================================================================
  */
 
-;(function(API) {
-'use strict'
+;
+(function (API) {
+	'use strict'
 
 /**
-Returns an array of length matching length of the 'word' string, with each
-cell occupied by the width of the char in that position.
-
-@function
-@param word {String}
-@param widths {Object}
-@param kerning {Object}
-@returns {Array}
-*/
+ * Returns an array of length matching length of the 'word' string, with each
+ * cell occupied by the width of the char in that position.
+ * 
+ * @function
+ * @param word {String}
+ * @param widths {Object}
+ * @param kerning {Object}
+ * @returns {Array}
+ */
 var getCharWidthsArray = API.getCharWidthsArray = function(text, options){
 
 	if (!options) {
 		options = {}
 	}
+  
```

Comment:
tests fail because there is an (merge?) error
```

var getCharWidthsArray = API.getCharWidthsArray = function(text, options){

	if (!options) {
		options = {}
	}
  
	var getCharWidthsArray = API.getCharWidthsArray = function (text, options) {

		if (!options) {
			options = {}
		}
```

Reply:
fix is #1501